commit f95cde60ff02e897088d35052426964a3db90eb7
Author: Chris Mayo <aklhfex@gmail.com>
Date:   Tue Jul 26 07:55:54 2016 +0100

build: add --htmldir option

Defaults to docdir but makes it possible to install html documentation
separately.

diff --git a/wscript b/wscript
index 9269d95..309efd3 100644
--- a/wscript
+++ b/wscript
@@ -942,6 +942,7 @@ _INSTALL_DIRS_LIST = [
     ('datadir', '${PREFIX}/share',    'data files'),
     ('mandir',  '${DATADIR}/man',     'man pages '),
     ('docdir',  '${DATADIR}/doc/mpv', 'documentation files'),
+    ('htmldir', '${DOCDIR}',          'html documentation files'),
     ('zshdir',  '${DATADIR}/zsh/site-functions', 'zsh completion functions'),

     ('confloaddir', '${CONFDIR}', 'configuration files load directory'),
diff --git a/wscript_build.py b/wscript_build.py
index 13d89ce..257be27 100644
--- a/wscript_build.py
+++ b/wscript_build.py
@@ -19,7 +19,7 @@ def _build_html(ctx):
         target       = 'DOCS/man/mpv.html',
         source       = 'DOCS/man/mpv.rst',
         rule         = '${RST2HTML} ${SRC} ${TGT}',
-        install_path = ctx.env.DOCDIR)
+        install_path = ctx.env.HTMLDIR)

     _add_rst_manual_dependencies(ctx)

